cloud-interfaces/api/cloudblockstorage-api.rst

```diff
@@ -159,10 +158,10 @@ In the sample response,
 correspond to the information available on the Cloud Control Panel.
 
 In the
-:rax-docs:`Getting Started Guide for the Cloud Block Storage API <cbs/api/v1.0/cbs-getting-started>`,
-you can see an example of
-:rax-docs:`obtaining a list of Cloud Block Storage volumes by using the cURL command-line interface (CLI)
-<cbs/api/v1.0/cbs-getting-started/content/Listing_volumes_d1e060.html>`.
+:rax-docs:`Getting Started Guide for the Cloud Block Storage API <cloud-block-storage/v1/getting-started/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing existing block storage volumes
+<cloud-block-storage/v1/getting-started/use-API-directly/#listing-existing-block-storage-volumes>`.
 
 Learn about Cloud Block Storage in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There are 2 broken links in this section to SDK QuickStart for Cloud Block Storage